src/messages/manifestjson.js

```diff
@@ -76,6 +76,15 @@ export const MANIFEST_BAD_OPTIONAL_PERMISSION = {
   file: MANIFEST_JSON,
 };
 
+export const MANIFEST_BAD_HOST_PERMISSION = {
+  code: 'MANIFEST_BAD_HOST_PERMISSION',
+  message: i18n._('The permission type is unsupported.'),
+  // TODO: File and reference here an issue to update this description with a link to the
```

Comment:
Is there an issue filed yet? Maybe we should add a link instead of a long comment here.

Reply:
@willdurand We don't have that doc page yet on MDN ([1]), and so this TODO was meant to remind me to file an issue in this repo, to follow up once we got a MDN page to link here.

I'm going to file the issue and update the patch to mention it here.

[1]: we have been recently discussing about MV3-related doc page on MDN, but we did not settle yet on the process we will use to gradually introduce them (e.g. I'm not yet sure if we are going to create these pages and land them while MV3 addons can't be yet submitted to AMO or if they will stay in a more long living branch in a [mdn/content](https://github.com/mdn/content) fork until we start accepting MV3 extensions on AMO)

Reply:
Updated inline comment to reference #3893.